src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -78,6 +78,44 @@ end
 SimpleDiGraph(nv::T, ne::Integer; seed::Int=-1) where T <: Integer =
     SimpleDiGraph{Int}(nv, ne, seed=seed)
 
+"""
+	SimpleDiGraph(g, p_in, p_out; seed=-1)
+
+Create a directed graph from an undirected graph `g` where the direction of each
+edge from `u` to `v` (`u < v`) in `g` is added as a directed edge from `u` to `v`
+with probability `p_out`, from `v` to `u` with probability `p_in`, or in both
+directions with probability `1 - p_in - p_out`. If `p_in + p_out ≥ 1`, `p_out`
+will be truncated to `1 - p_in`, and no bidirectional edges will be created.
+"""
+function SimpleDiGraph(g::SimpleGraph, _p_in::Real, _p_out::Real; seed::Int=-1)
```

Comment:
I am still not convinced, that a constructor is the right thing to use here. It does not give any hints to someone reading the code using this, what this function does. Furthermore, I don't know how much this function will be used, so it will make it impossible to overload this method signature in the future.

If you still want to use the constructor, then I think it should at least redirect to another function, like we do for erdos_renyi.

Reply:
I'm ok with calling it something else. What do you suggest?